client/my-sites/importer/importing-pane.jsx

```diff
@@ -193,6 +196,10 @@ class ImportingPane extends React.PureComponent {
 		return this.isInState( appStates.MAP_AUTHORS );
 	};
 
+	startImporting = () => {
+		this.props.startImporting( this.props.importerStatus );
+	};
```

Comment:
fun fact! we can skip the clutter with a single expression

```js
startImporting = () => this.props.startImporting( this.props.importerStatus );
```

Reply:
👍 will shorten.